Sources/JWTProvider/JWTProviderError.swift

```diff
@@ -4,12 +4,13 @@ import HTTP
 public enum JWTProviderError: Error {
     case noVerifiedJWT
     case noJWTSigner
+    case noJWKSURL
```

Comment:
Adding cases to an enum is a breaking change. We'll have to add a new error type or use one of the existing ones.